### PR TITLE
Move instructions for shell completion from Wiki into docs

### LINKED ISSUE #147: Move instructions for shell completion from Wiki into docs
Just to keep all things in a single place &mdash; when anything changes wrt. how the completion is done, we should reflect it in the docs in unison.

### PR DESCRIPTION
**Pull request by mohitsaxenaknoldus**
Closes #147

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +38,98 @@ a port into a plugin for the IntelliJ Platform products, including PyCharm, WebS
...
+If `git` resolves to `/usr/bin/git`, then likely `git` is the default installation provided in Mac OS.
+As a workaround, add the following line directly at the end of `~/.zshrc`:
+```shell script
+source ~/.oh-my-zsh/custom/plugins/git-machete/git-machete.plugin.zsh
+```
+and reload the shell.
+
+If `git` resolves to `/usr/local/bin/git`, then likely `git` has been installed via `brew`.
+Up to our current knowledge, workaround is much harder to provide in such scenario.
+One option is to `brew uninstall git` and then use the solution for Mac's default `git` provided above,
```
Comment (PawelLipski) [inline README.md]: Pls add newline before this line

File: README.md [hunk 2]
```diff
@@ -38,8 +38,98 @@ a port into a plugin for the IntelliJ Platform products, including PyCharm, WebS
...
+but that's likely undesired since `git` shipped with Mac OS is almost always an older version than what's available via `brew`.
+Another, less intrusive workaround is to make sure that the zsh `_git` function
```
Comment (PawelLipski) [inline README.md]: Pls add newline before this line

File: README.md [hunk 3]
```diff
@@ -38,8 +38,98 @@ a port into a plugin for the IntelliJ Platform products, including PyCharm, WebS
 
 We suggest a couple of alternative ways of installation.
 
-**Bash and zsh completion scripts are provided** in completion/ directory,
-see [wiki for their installation instructions](https://github.com/VirtusLab/git-machete/wiki).
+### Bash
```
Comment (PawelLipski) [inline README.md]: Actually... I'd rather avoid putting all these details directly into README, esp. before Installation and Quick start sections which are far more important for the newcomers :/

Comment (PawelLipski) [reply README.md]: I rather thought about putting that manual into a separate file within this repository (as opposed to wiki, which is technically a separate repo) and updating the link in README... how about `completion/README.md`?

File: README.md (hunk 1 above)
Comment (mohitsaxenaknoldus) [reply README.md:95]: Above line 95?

File: README.md (hunk 3 above)
Comment (mohitsaxenaknoldus) [reply README.md]: yeah sure! Check now please.

File: README.md [hunk 4]
```diff
@@ -38,8 +38,7 @@ a port into a plugin for the IntelliJ Platform products, including PyCharm, WebS
 
 We suggest a couple of alternative ways of installation.
 
-**Bash and zsh completion scripts are provided** in completion/ directory,
-see [wiki for their installation instructions](https://github.com/VirtusLab/git-machete/wiki).
+**Bash and zsh completion scripts are provided in [completion/README.md](completion/README.md)**
```
Comment (PawelLipski) [inline README.md]: For clarity:
```suggestion
**Instructions for installing bash and zsh completion scripts are provided in [completion/README.md](completion/README.md).**
```

**Review by PawelLipski — APPROVED** [review]
Small fix left, otherwise approving ‼️

**Comment by mohitsaxenaknoldus** [comment]
> Small fix left, otherwise approving

Fixed! Thanks.

**Pull request merged**